x-pack/plugins/apm/public/hooks/use_environments_fetcher.tsx

```diff
@@ -34,7 +40,7 @@ export function useEnvironmentsFetcher({
   start?: string;
   end?: string;
 }) {
-  const { data = INITIAL_DATA, status } = useFetcher(
+  const { data = INITIAL_DATA, status = FETCH_STATUS.LOADING } = useFetcher(
```

Comment:
why do we need a default value here?

Reply:
hmm, it was there before I just updated the hardcoded `status = "loading"` with the constant but I believe we don't need it